Description: aarch64: define __clone, make clone weak
 libpthread requires an internal __clone to be defined and clone to be
 a weak alias against it. Otherwise linking against -lpthread would
 fail in many cases.
Author: Christian Seiler <christian@iwakd.de>
Last-Update: 2017-01-05

--- a/aarch64/clone.S
+++ b/aarch64/clone.S
@@ -3,7 +3,8 @@
 
 #include <errno.h>
 
-FUNC_START	clone
+FUNC_START_WEAK	clone
+FUNC_START	__clone
 	/* sanity checks */
 	cbz	x0, .Linval
 	cbz	x1, .Linval
@@ -26,4 +27,5 @@ FUNC_START	clone
 	ldp	x1, x0, [sp], #16
 	blr	x1
 	b	exit
+FUNC_END	__clone
 FUNC_END	clone
